paper-tabs.html

```diff
@@ -285,6 +285,27 @@
           value: 'paper-tab'
         },
 
+        /**
+         * If true, tabs are not automatically selected when focused using the
+         * keyboard.
+         */
+        noAutoselect: {
```

Comment:
Why are we adding this property? I'm not sure I understand from the description. Did anybody ask for it?